.ci_support/linux_.yaml

```diff
@@ -1 +1,4 @@
-{}
+hdf5:
+- 1.10.1
+openblas:
+- 0.2.20
```

Comment:
Awesome! This is what I wanted to see :smile: 

Thanks @msarahan for explaining the `{{ pin_compatible('hdf5', max_pin='x.x.x') }}`.

@ChristopherHogan this is all new and we are in the process of writing docs, sorry for making your PR the guinea pig :grimacing: